iflist.go

```diff
@@ -67,16 +67,18 @@ func parseInterfaces(content []byte) *InterfaceList {
 	output := string(content)
 	lines := strings.Split(output, "\n")
 
+	interfaceRegex := regexp.MustCompile(`\*INTERFACES\*`)
+	routesRegex := regexp.MustCompile(`\*ROUTES\*`)
```

Comment:
Usually `regexp.MustCompile` statements are kept outside of the function body, in `var` declarations at the top of the files they are used in. WDYT?

See [discussion about it in the noglobals linter if you're curious](https://github.com/leighmcculloch/gochecknoglobals/issues/6)